code/game/objects/items/explosives/mine.dm

```diff
@@ -172,7 +172,8 @@
 			return
 		else
 			..()
-			use_dir = FALSE // Claymore defaults to radial in these case. Poor man C4
+			// Claymore defaults to radial in these case. Poor man C4
+			use_dir = TRUE
```

Comment:
claymore are not supposed to use DIR if they dont have the right assembly . rever this